inst/WORDLIST

```diff
@@ -1,9 +1,32 @@
+args
+brushedPoints
+Closeable
+closeable
+desc
 Draggable
+draggable
+dt
 Forkers
+funder
+ggplot
+github
 Hoffmann
+href
+https
+insightsengineering
+io
+jQuery
```

Comment:
@vedhav can you tick out `jQuery` in your documentation so that we do not need to add this to WORDLIST?